### PR TITLE
Add model validations and specs

### PR DESCRIPTION
**Pull request by SergSkol**
Add the following validations:
-For the User model:
--Name must not be blank.
--PostsCounter must be an integer greater than or equal to zero.
-For the Post model:
--Title must not be blank.
--Title must not exceed 250 characters.
--CommentsCounter must be an integer greater than or equal to zero.
--LikesCounter must be an integer greater than or equal to zero.
-Add unit specs for all of your models' methods and validations.

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -3,6 +3,10 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :title, presence: true, length: { maximum: 250 }
```
Comment (FrederickMih) [inline app/models/post.rb:6]: - [ ] Hi @SergSkol, what do you think of kindly adding a test for the scenario where the title of the post must not exceed 250 characters?

File: spec/post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  it 'CommentsCounter must be an integer greater than or equal to zero.' do
+    subject.comments_counter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'LikesCounter must be an integer greater than or equal to zero.' do
+    subject.likes_counter = -1
+    expect(subject).to_not be_valid
+  end
+end
```
Comment (FrederickMih) [inline spec/post_spec.rb]: - [ ] I equally realized that most of the tests are failing as you can see in the screenshot below. Please let's kindly fix :wrench:  those issues.


Screenshot from 2023-03-02 10-11-26

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand:  @SergSkol, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights :green_circle: 

- PR is professional :+1: 
- Descriptive README.md file :+1: 
- Linter checks passed :heavy_check_mark: 


## Required Changes ♻️

- [ ] @SergSkol, you'll kindly be required to add unit and validation tests for all your models as requested in the requirements. You have done a fantastic job adding validation tests for the User and Post models :+1: . We are left with `Comment` and `Like` models

_Check the comments under the review._

## Optional suggestions
**N/A**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — APPROVED** [review]
### Hi @SergSkol 

Great job implementing the recommended changes by the previous reviewer.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ] [OPTIONAL] Kindly update your PR description. The PR description should contain a summary of changes that took place in the project.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**